bindings/c/include/kllvm-c/kllvm-c.h

```diff
@@ -0,0 +1,68 @@
+#ifndef KLLVM_C_H
+#define KLLVM_C_H
+
+#ifndef __cplusplus
+#include <stdbool.h>
+#endif
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/* Opaque types */
+
+typedef struct kore_pattern kore_pattern;
+typedef struct kore_sort kore_sort;
+typedef struct kore_symbol kore_symbol;
+typedef struct block block;
+
+/* KOREPattern */
+
+char *kore_pattern_dump(kore_pattern const *);
+
+void kore_pattern_free(kore_pattern const *);
+
+kore_pattern *kore_pattern_new_token(char const *, kore_sort const *);
+kore_pattern *kore_pattern_new_injection(
+    kore_pattern *, kore_sort const *, kore_sort const *);
+kore_pattern *kore_pattern_make_interpreter_input(kore_pattern *);
```

Comment:
what's the purpose of this function? do i use it in conjunction with kore_simplify_bool?

Reply:
It's not needed for what you're doing currently, but you might want it in the future if this binding needs to do actual rewrite steps rather than just evaluating functions of sort Bool.

The function essentially mimics what llvm-krun does in terms of composing a call to `initGeneratedTopCell` from a config variable map.